dev-envs/linux/default-vscode-extensions.txt

```diff
@@ -13,6 +13,5 @@ redhat.vscode-xml
 redhat.vscode-yaml
 rust-lang.rust-analyzer
 timonwong.shellcheck
-twxs.cmake
```

Comment:
In the past few months this became deprecated and is now part of the main cmake extension